jans-config-api/plugins/scim-plugin/src/main/java/io/jans/configapi/plugin/scim/model/config/ScimConf.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Janssen Project software is available under the MIT License (2008). See http://opensource.org/licenses/MIT for full text.
+ *
+ * Copyright (c) 2020, Janssen Project
+ */
+
+package io.jans.configapi.plugin.scim.model.config;
+
+import io.jans.as.model.config.StaticConfiguration;
+import io.jans.orm.annotation.AttributeName;
+import io.jans.orm.annotation.DN;
+import io.jans.orm.annotation.DataEntry;
+import io.jans.orm.annotation.JsonObject;
+import io.jans.orm.annotation.ObjectClass;
+
+@DataEntry
+@ObjectClass(value = "jansAppConf")
+public class ScimConf {
+
+    @DN
+    private String dn;
+
+    @JsonObject
+    @AttributeName(name = "jansConfDyn")
+    private ScimAppConfiguration dynamicConf;
+
+    @JsonObject
+    @AttributeName(name = "jansConfStatic")
+    private StaticConfiguration staticConf;
+
+    @AttributeName(name = "jansRevision")
+    private long revision;
+
+    public String getDn() {
+        return dn;
+    }
+
+    public void setDn(String p_dn) {
```

Comment:
Sonar does not like hungarian notation. It increase sonar smell. Lets use `this.dn = dn;` (I know we have many such on AS side, it has to be corrected there as well to improve statistic.)

Reply:
Resolved